src/services/DiscordService.ts

```diff
@@ -0,0 +1,200 @@
+import profiles from 'decentraland-gatsby/dist/utils/loader/profile'
+import { ChannelType, Client, EmbedBuilder, GatewayIntentBits } from 'discord.js'
+
+import { capitalizeFirstLetter } from '../clients/utils'
+import { getProfileUrl } from '../entities/Profile/utils'
+import { ProposalType } from '../entities/Proposal/types'
+import { isGovernanceProcessProposal, proposalUrl } from '../entities/Proposal/utils'
+import UpdateModel from '../entities/Updates/model'
+import { UpdateAttributes } from '../entities/Updates/types'
+import { getPublicUpdates, getUpdateNumber, getUpdateUrl } from '../entities/Updates/utils'
+
+const CHANNEL_ID = process.env.DISCORD_CHANNEL_ID
+const TOKEN = process.env.DISCORD_TOKEN
+
+const client = new Client({ intents: [GatewayIntentBits.Guilds] })
+
+const DCL_LOGO = 'https://decentraland.org/images/decentraland.png'
+const DEFAULT_AVATAR = 'https://decentraland.org/images/male.png'
+const BLANK = '\u200B'
+const PREVIEW_MAX_LENGTH = 140
+
+type Field = {
+  name: string
+  value: string
+}
+
+enum MessageColors {
+  NEW_PROPOSAL = 0x0099ff,
+  FINISH_PROPOSAL = 0x8142f5,
+  NEW_UPDATE = 0x00ff80,
+}
+
+type EmbedMessageProps = {
+  title: string
+  proposalType?: ProposalType
+  description?: string
+  fields: Field[]
+  user?: string
+  action: string
+  color: MessageColors
+  url: string
+}
+
+function getChoices(choices: string[]): Field[] {
+  return choices.map((choice, idx) => ({
+    name: `Option #${idx + 1}`,
+    value: capitalizeFirstLetter(choice),
+  }))
+}
+
+function shortText(text: string) {
+  return text.length > PREVIEW_MAX_LENGTH ? text.slice(0, PREVIEW_MAX_LENGTH) + '...' : text
+}
+
+export class DiscordService {
+  static init() {
```

Comment:
Does anything guarantee us that this is called every time we create a DiscordService instance? 
Maybe it'd be better if we had this as the constructor?

Reply:
This method is only called once on `server.ts`. And I think it is better that way to control exactly when we want to call it. We don't create instances of this class but use its methods. So it should always use the same client instance.